src/harness/loggedIO.ts

```diff
@@ -159,7 +159,7 @@ namespace Playback {
     }
 
     const canonicalizeForHarness = ts.createGetCanonicalFileName(/*caseSensitive*/ false); // This is done so tests work on windows _and_ linux
-    function sanitizeTestFilePath(name: string) {
+    export function sanitizeTestFilePath(name: string) {

```

Comment:
`src/harness/harness.ts` has a copy of this function - now that this is exported, could you make it point at this copy and remove the duplicate (probably should have done that from the beginning)?